src/test_utils/dl_string_extractor/dl_string_parsing_test_driver.cc

```diff
@@ -0,0 +1,41 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-----------------------------------------------------------------------------
+//
+// This file is a simple driver to run datalog scripts generated from
+// extracting raw strings representing parts of a datalog program to ensure
+// that they parse correctly.
+//
+#include <memory>
+
+#include "souffle/SouffleInterface.h"
+
+int main(int argc, char **argv) {
```

Comment:
Not for this PR, but we should consolidate the different test drivers into one place (or even one binary). I have filed #160. Can you add a TODO here?

Reply:
Yes, sounds good.

Reply:
Done.